fusequery/query/src/servers/mysql/mysql_interactive_worker.rs

```diff
@@ -93,7 +93,13 @@ impl<W: std::io::Write> MysqlShim<W> for InteractiveWorker<W> {
         let context = self.session.create_context();
 
         context.attach_query_info(query);
-        DFQueryResultWriter::create(writer).write(self.base.do_query(query, context))?;
+        match DFQueryResultWriter::create(writer).write(self.base.do_query(query, context)) {
```

Comment:
Maybe to use `if let Err(cause) = DFQueryResultWriter::create(writer).write(self.base.do_query(query, context))` is better(because we don't care Ok?)

Reply:
Yes sir